### DISCUSSION TITLE
Osu not opening after selecting opengl

### DISCUSSION THREAD

**Discussion by MarkoVityk** [ppy/osu#29053] [Q&A] [answered]
### Type

Other

### Bug description

I was watching BTMC's twitch when he said Opengl was better, so I tried it. (note: directx11 was working perfectly fine) After I closed it, it wouldn't open.

### Version

newest one as of this video: https://www.youtube.com/watch?v=e8tKtedQvYc

### Logs

[1721874182.runtime.log](https://github.com/user-attachments/files/16369476/1721874182.runtime.log)
opengl (up)
vs
directx11 (down)
[1721759681.runtime.log](https://github.com/user-attachments/files/16369489/1721759681.runtime.log)

**Comment by smoogipoo** [comment, marked as answer]
Please don't cherry pick the logs and just upload everything...

Hard to say what is broken here, because of the above, but you can go into the `framework.ini` file and remove the `Renderer` line.
